pkg/machines/libvirt-dbus.es6

```diff
@@ -654,6 +655,30 @@ LIBVIRT_DBUS_PROVIDER = {
         };
     },
 
+    SET_MEMORY ({
+        name,
+        id: objPath,
+        connectionName,
+        memory
+    }) {
+        const updatedMemory = memory * 1024;
```

Comment:
Since you need kilobytes here why you don't pass the value in kilobytes from the react code and you transform in mb and then kb?